src/solver_interface.jl

```diff
@@ -218,6 +218,14 @@ the model `m`.
 function delete_linear_constraints! end
 @deprecate lqs_delrows! delete_linear_constraints!
 
+"""
+    delete_quadratic_constraints!(m, start_row::Int, end_row::Int)::Void
+
+Delete the quadratic constraints `start_row`, `start_row+1`, ..., `end_row` from
+the model `m`.
+"""
+function delete_quadratic_constraints! end
```

Comment:
@joaquimg I added this method to the API

Reply:
Nice! Are there MOI tests for deleting quadratics?

Reply:
Yip. 

See https://github.com/JuliaOpt/MathOptInterface.jl/pull/354

As well as https://github.com/JuliaOpt/Gurobi.jl/pull/125